packages/@lwc/template-compiler/src/config.ts

```diff
@@ -35,7 +35,6 @@ export interface ResolvedConfig extends Required<Config> {
 }
 
 const AVAILABLE_OPTION_NAMES = new Set([
-    'secure',
```

Comment:
agreed! I don't see the option used anywhere. Is this the only code we can remove?